docs/templating/markdown.html

```diff
@@ -102,14 +102,13 @@ <h4>Currently in Beta - <a href="../../old-learn.html">View old docs</a></h4>
     <div id="doc-container" class="docs-item-4 docs-window">
       <main>
         <h1 class="heading-1">Kitura Markdown</h1>
-        <h2 class="heading-2">Prerequisites</h2>
-        <ul class="plain-list">
-          <li><a onclick="setActiveSidebarElementForParent('create-server')" href="../getting-started/create-server.html#">Kitura Server:</a> Learn how to create one in our Getting Started guide.</li>
-          <li><a onclick="localStorage.setItem('package', 'Kitura-Markdown'); setActiveSidebarElementForParent('markdown')" href="../getting-started/update-package.html#">Update Dependencies:</a> KituraMarkdown needs adding to the `Package.swift` file.</li>
-        </ul>
-        <div class="underline"></div>
+        <p>To serve a web page generated from <a target="_blank" href="https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet"> Markdown templates</a> (.md files) we use <a target="_blank" href="https://github.com/IBM-Swift/Kitura-Markdown"> Kitura-Markdown</a>.</p>
```

Comment:
Can we have some more words here please, something like... "In this guide, we will show you how to serve a web page generated from Markdown templates (.md files) using Kitura-Markdown." with all the relevant links included.